contracts/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@bisonai/orakl-contracts",
+  "version": "2.4.0",
+  "description": "The Orakl Network Smart Contracts",
+  "files": [
+    "v0.1/dist",
+    "v0.1/src",
+    "v0.1/package.json",
+    "v0.2/dist",
+    "v0.2/src",
+    "v0.2/package.json"
+  ],
+  "keywords": [
+    "Oracle",
+    "Orakl",
+    "Orakl Network",
+    "Blockchain",
+    "Web3",
+    "VRF",
+    "Request-Response",
+    "Data Feed"
+  ],
+  "engines": {
+    "node": ">=20.0.0"
+  },
+  "author": "Bisonai",
+  "homepage": "https://github.com/Bisonai/orakl/tree/master/contracts#orakl-network-contracts",
+  "license": "MIT"
+}
```

Comment:
(Optional) How about adding following script command? Both folder would have to be added into yarn workspaces
```
  "scripts": {
    "compile":  "yarn --cwd='../' contracts-v01 compile && yarn --cwd='../' contracts-v02 compile"
    "build": "yarn --cwd='../' contracts-v01 build && yarn --cwd='../' contracts-v02 build"
    "test": "yarn --cwd='../' contracts-v01 test && yarn --cwd='../' contracts-v02 test"
  }
```
Through the command, both v0.1 and v0.2 contracts test could be built and tested in github action. If you think it is out of scope from current pr, it could be updated from further pr